Spec/README.md

```diff
@@ -507,8 +507,6 @@ Each file/dir has the following purpose:
 
 **[documentation](https://github.com/Omnibus-Type/Texturina/tree/master/documentation)**: Directory which contains informations about the Family. You can eventually store there your specimen, the pictures you use for the README.md etc.
 
-**[DESCRIPTION.en_us.html](https://github.com/Omnibus-Type/Texturina/blob/master/documentation/DESCRIPTION.en_us.html):** A small html snippet which describes the family. The text should be concise. This file is used on the main Google Fonts website for each family's "About" section e.g https://fonts.google.com/specimen/Oswald. 
-
 This file must include: 
```

Comment:
This section should also be removed since it's related to the description.

Reply:
didn't I remove it ?

Reply:
You removed L510-L511 which is a summary, but L512-L522 are related too.

Reply:
Ah right, I moved it.